numpy/core/src/umath/loops.c.src

```diff
@@ -786,6 +795,14 @@ NPY_NO_EXPORT void
     }
 }
 
+NPY_NO_EXPORT void
+@TYPE@_isfinite(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    OUTPUT_LOOP {
+        *((@type@ *)op1) = 1;
```

Comment:
From a user perspective, I would usually prefer to be able to use a function like `np.isfinite` (or maybe `np.isna`) that does not require me to know the dtype of the array ahead of time. But `np.isnat` could also be useful in some cases.